spec/controller/posts_controller_spec.rb

```diff
@@ -4,7 +4,7 @@
 
 RSpec.describe PostsController, type: :controller do
   let(:user) { create(:user) }
-  let(:first_post) { create(:post) }
+  let(:first_post) { create(:post, user:) }
```

Comment:
?

Reply:
Here I just add association user with post, but failed the test

Reply:
why do you pass empty paramether user to create?

Reply:
all of my comments was related to that